### DISCUSSION TITLE
Welcome to ribot Discussions!

### DISCUSSION THREAD

**Discussion by alberto-abarzua** [alberto-abarzua/3d_printed_robot_arm#58] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by moldmet** [comment]
Hello
I ran into a problem during the software installation. When I run the simulation, everything seems fine. I see the Robotmodel at localhost:8000. But when I open localhost:3000 I  get the Message 'Prepping!' and then the window becomes white and I get nothing to see. The first thing I tried was to swap from Chrome to Firefox. But with the same result. What am I doing wrong?
